robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentingClassLoaderConfig.java

```diff
@@ -35,48 +36,106 @@
 import java.util.ServiceLoader;
 import java.util.Set;
 
-import static java.util.Arrays.asList;
-
 /**
  * Configuration rules for {@link org.robolectric.internal.bytecode.InstrumentingClassLoader}.
  */
 public class InstrumentingClassLoaderConfig {
```

Comment:
I was thinking to rename this InstrumentationConfig or InstrumentationConfigutation the name sounds a little more fluid.